### PR TITLE
Add discourse to social icons

### PR DESCRIPTION
**Pull request by joshka**
#### Description

Forum software - https://discourse.org/

The middle icon is the new icon (in the context of https://github.com/ratatui-org/ratatui-website/)
image

### CODE REVIEW

File: packages/starlight/components/Icons.ts
Comment (delucis) [inline packages/starlight/components/Icons.ts:149]: Ran this through [SVGOMG](https://jakearchibald.github.io/svgomg/) to squeeze it a little bit smaller (~50 bytes gzipped):

```suggestion
	discourse: '<path d="M12.102 0h-.081C5.462 0 .13 5.252.001 11.779v.012L.007 24l12.097-.01c6.582-.055 11.897-5.404 11.897-11.995S18.686.056 12.109 0h-.005zM12 18.857h-.015a6.778 6.778 0 0 1-2.94-.666l.041.018-4.345 1.077 1.227-4.018a6.78 6.78 0 0 1-.83-3.27A6.86 6.86 0 1 1 12 18.857z"/>'
```

**Review by delucis — COMMENTED** [review]
Thank you for the contribution @joshka! Have one small suggestion.

I might also move the new `discourse` entry in `Icons.ts` up to be with the other social icons although we don’t have a strict rule about this, i.e. I guess immediately after `blueSky` would make sense to keep social platforms roughly grouped together.

**Comment by joshka** [comment]
> I might also move the new `discourse` entry in `Icons.ts` up to be with the other social icons although we don’t have a strict rule about this, i.e. I guess immediately after `blueSky` would make sense to keep social platforms roughly grouped together.

done in https://github.com/withastro/starlight/pull/1909/commits/39de623676f8c162091f4838ea7d19a2cf6773cf

**Review by delucis — APPROVED** [review]
Thank you @joshka — looks great 🙌 

Will merge this when we’re ready to cut our next patch release. (Merged PRs go live in our docs immediately, so we hold off until just before release to avoid someone seeing a feature in the docs that isn’t actually available yet.)

**Pull request merged**
